docs/usage_and_configuration/ways_to_install_black.md

```diff
@@ -0,0 +1,46 @@
+# Ways to install Black
+
+There are several ways you can install _Black_.
+
+## Install from PyPI
+
+To install the latest release of _Black_ from [PyPI] (Requires Python 3.8+), use:
+
+`pip install black`
+
+A _Black_ release currently offers three types of artifacts via PyPI, as outlined in the
+[Release Process]:
+
+1. The source distribution of the release
+2. Generic Python wheel, meant for use on any Python supported platform
+3. Platform and Python version specific wheels that offer significantly improved
+   performance, compiled using [mypyc]
+
+By default, `pip` will prefer a compatible wheel and revert to the source distribution
+if no such wheels are found, as outlined in [Python documentation].
```

Comment:
```suggestion
if no such wheels are found, as outlined in [Python documentation](LINK).
```
Did you forget the link?

Reply:
The links were all declared at the end of the document, similar to other pages in the documentation. I've changed all of them to avoid confusion.